akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceSpec.scala

```diff
@@ -27,6 +27,15 @@ class SourceSpec extends StreamSpec with DefaultTimeout {
   implicit val config = PatienceConfig(timeout = Span(timeout.duration.toMillis, Millis))
 
   "Single Source" must {
+
+    "produce exactly one element" in {
+      //#source-single
+      val s: Future[immutable.Seq[Int]] = Source.single(1).runWith(Sink.seq)
+      s.futureValue should ===(immutable.Seq(1))
```

Comment:
I think this unnecessarily pulls in the ScalaTest DSL into the sample, maybe like my suggestion for java here as well `s.foreach(list => println(s"collected elements: $list"))`, and then move the assert outside of the sample markers,